tests/backup/backup_rancher_test.go

```diff
@@ -754,6 +767,27 @@ var _ = Describe("{MultipleProjectsAndNamespacesBackupAndRestore}", func() {
 			err = CreateRestoreWithProjectMapping(restoreName, backupList[0], namespaceMapping, destinationClusterName, orgID, ctx, storageClassMapping, projectUIDMapping, projectNameMapping)
 			dash.VerifyFatal(err, nil, fmt.Sprintf("Creating restore- %s in different project and different namespace with storage class mapping from backup %s in destination cluster", restoreName, backupList[0]))
 		})
+
+		Step("Remove the source cluster namespaces from the project", func() {
+			log.Infof("Remove the source cluster namespaces from the project")
+			err = Inst().S.(*rke.Rancher).RemoveNamespaceFromProject(appNamespaces)
+			dash.VerifyFatal(err, nil, fmt.Sprintf("Removing the source cluster namespaces %s from the project", appNamespaces))
+		})
+
+		Step("Taking backup of namespaces after moving them to no project", func() {
+			log.InfoD("Taking backup of namespaces %s after moving them to no project", appNamespaces)
+			noProjectBackup = fmt.Sprintf("%s-%v-no-project", BackupNamePrefix, RandomString(10))
+			appContextsToBackup := FilterAppContextsByNamespace(scheduledAppContexts, appNamespaces)
+			err = CreateBackupWithValidation(ctx, noProjectBackup, SourceClusterName, customBackupLocationName, backupLocationUID, appContextsToBackup, nil, orgID, sourceClusterUid, "", "", "", "")
+			dash.VerifyFatal(err, nil, fmt.Sprintf("Creation and validation of backup [%s] with the namespaces %s after removing the namespaces from project", noProjectBackup, appNamespaces))
+		})
+		Step("Restore the backup taken after all the namespaces are removed from the project", func() {
+			log.InfoD("Restore the backup taken after all the namespaces are removed from the project")
+			restoreName := fmt.Sprintf("%s-%v-no-project", restoreNamePrefix, RandomString(10))
+			err = CreateRestoreWithValidation(ctx, restoreName, noProjectBackup, make(map[string]string), make(map[string]string), destinationClusterName, orgID, scheduledAppContexts)
+			dash.VerifyFatal(err, nil, fmt.Sprintf("Creating restore [%s] from backup [%s]", restoreName, noProjectBackup))
+			restoreList = append(restoreList, restoreName)
+		})
```

Comment:
How about including project name in the log messages?

Reply:
Which project name you are asking to include in the log?
Here all the namespaces now will not be under any project, that is the intention of the newly added step
@kphalgun-px

Reply:
Oh, I see. I did not fully grasp what is going on. I thought the project name was missing. Thanks for clarification.